fallout/turfs/plating.dm

```diff
@@ -257,32 +257,51 @@
 /turf/open/floor/plating/ground/road
 	name = "\proper road"
 	desc = "A stretch of road."
-	icon = 'fallout/icons/turf/roadsidewalk.dmi'
-	icon_state = "road"
-	var/dir_variation = TRUE
+	icon = 'fallout/icons/turf/road.dmi'
+	icon_state = "road-255"
+	base_icon_state = "road"
+	smoothing_flags = SMOOTH_BITMASK
+	smoothing_groups = list(SMOOTH_GROUP_FALLOUT_ROAD)
+	canSmoothWith = list(SMOOTH_GROUP_FALLOUT_ROAD)
 
 /turf/open/floor/plating/ground/road/Initialize()
 	. = ..()
-	if(dir_variation)
-		dir = pick(GLOB.cardinals)
+	addtimer(CALLBACK(src, /atom/.proc/update_icon), 1)
 
-/turf/open/floor/plating/ground/road/curb
-	icon_state = "curb"
-	dir_variation = FALSE
+/turf/open/floor/plating/ground/road/update_icon()
+	var/crack_randomiser = "crack_[rand(1,18)]"
+	var/road_randomiser = "rand(-10,10)"
+	var/direction_randomiser = "rand(0,8)"
+	if(prob(20))
+		add_overlay(image('fallout/icons/turf/road.dmi', crack_randomiser, FLOAT_LAYER, direction_randomiser, road_randomiser, road_randomiser))
+		return
```

Comment:
```suggestion

```
You are at the end of the proc and not really returning anything.
This is not necessary and does not add clarity.